tools/retry_autopkgtest.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# Retry the autopkgtest run in a pull request.
+# Arguments:
+#   release:      The name of the Ubuntu release to test.
+#   architecture: The architecture to test.
+#   pr_id:        The identifier of the pull request to test.
+# Environment variables:
+#   SNAPCRAFT_AUTOPKGTEST_SECRET: The secret to authenticate the test execution.
+
+if [ -z "${SNAPCRAFT_AUTOPKGTEST_SECRET}" ]; then
+    echo 'Set the secret to the environment variable $SNAPCRAFT_AUTOPKGTEST_SECRET.'
+    exit 1
+fi
+
+if [ "$#" != 3 ]; then
+    echo "Usage: "$0" release architecture pr_id"
```

Comment:
It would be nice to actually do it like this:

`$0 <PR> release:arch...`

So then we can call it once with

$0 889 zesty:amd64 yakkety:amd64 xenial:amd64